src/pages/[platform]/build-a-backend/storage/storage-v5-to-v6-migration-guide/index.mdx

```diff
@@ -0,0 +1,1265 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Migrate from v5 to v6',
+  description: 'Learn more about the migration steps to upgrade Storage APIs for Amplify JavaScript v5 to v6',
+  platforms: ['angular', 'javascript', 'nextjs', 'react', 'vue', 'react-native']
+};
+
+export const getStaticPaths = async () => {
+  return getCustomStaticPath(meta.platforms);
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+This guide will help you migrate v5 Storage implementations to the new functional API's in v6.
+
+## Deprecated Options
+
+<Callout>Many input and output properties have been deprecated due to low usage or added complexity. If you would like to see these options re-implemented in v6, feel free to open a [feature request](https://github.com/aws-amplify/amplify-js/issues/new?assignees=&labels=pending-triage&projects=&template=2.feature_request.yaml) in our github repository.</Callout>
+
+The input options below have been removed in v6 from Storage API's:
+
+- `provider`: _Custom providers are no longer supported in v6._
+- `customPrefix`: _You can [add a prefix resolver](/[platform]/build-a-backend/storage/configure-access/#customize-object-key-path) to your Storage Config to customize the key prefix per accessLevel. See the [Using a Custom Prefix](/[platform]/build-a-backend/storage/storage-v5-to-v6-migration-guide/#using-a-custom-prefix) header for migration details._
+- `track`: _Tracking Storage events in Hub is not yet supported in v6._
+- `ServerSideEncryption`, `SSECustomerAlgorithm`, `SSECustomerKey`, `SSECustomerKeyMD5`, and `SSEKMSKeyId`: _These options have been deprecated due to low usage and added complexity: Server-Side Encryption has been automatically applied using default keys by S3 [since Jan 2023](https://aws.amazon.com/blogs/aws/amazon-s3-encrypts-new-objects-by-default/)._
+- `cacheControl`: _This option has been removed and now defers to the HTTP caching behavior configured by the runtime._
+- `$metadata`: _Information like `httpStatusCode` and `attempts` will be provided in an upcoming release via debug messages using [the Console Logger](/[platform]/build-a-backend/utilities/console-logger/)._
```

Comment:
I think this might not be tied specifically to the Console Logger but a Logging category/feature